.github/workflows/ci.yml

```diff
@@ -129,6 +129,15 @@ jobs:
         run: |
           cd projects/plio-frontend/
 
+          # delete the node_modules created by the docker
+          echo 'deleting node modules'
+          rm -rf node_modules/
```

Comment:
When building the docker image, it does npm install and creates node_modules in it's own separate memory right? It doesn't create a node_modules in our app's folder no?

Maybe I'm confused